pkg/deploy/dev-workspace-config/dev_workspace_config.go

```diff
@@ -194,6 +196,14 @@ func updateWorkspaceImagePullPolicy(imagePullPolicy corev1.PullPolicy, workspace
 	workspaceConfig.ImagePullPolicy = string(imagePullPolicy)
 }
 
+func updateIgnoredUnrecoverableEvents(operatorConfig *controllerv1alpha1.OperatorConfiguration, ignoredUnrecoverableEvents []string) {
+	if ignoredUnrecoverableEvents == nil {
```

Comment:
Why do we need this nil check? Can't we just do `operatorConfig.Workspace.IgnoredUnrecoverableEvents = ignoredUnrecoverableEvents`?

Reply:
I agree. I removed the nil check.